docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+
+  db:
+    image: postgres
+    restart: always
+    shm_size: 128mb
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres_password
```

Comment:
I am not recommending to push the password to github. 
Best would be to insert it from the env variables and put a placeholder with a dollar sigh here.